extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -612,6 +616,9 @@ export class Carousel {
    * @param {!Array<!Element>} slides
    */
   updateSlides(slides) {
+    if (!slides.length) {
+      dev().warn(TAG, 'No slides were found.');
```

Comment:
Won't this warn for `amp-base-carousel` even if the component with no slides is `amp-carousel` or `amp-stream-gallery`?

Reply:
Yes that is the idea. Is there a valid situation in which `amp-stream-gallery` will update it's slides with an empty array?

Reply:
I think this will be a confusing message primarily for users of `amp-carousel`. I am not too worried about `amp-stream-gallery` since 0.1 is still experimental.

Reply:
Talked offline, to avoid confusing warning message, let's use the element's tag name as the TAG parameter, so that developers know where the warning is coming from.